app/dashboard/poh_utils.py

```diff
@@ -2,6 +2,15 @@
 
 _poh_contract = None
 
+EIP_1271_ABI = [{"constant":True,"inputs":[{"name":"_messageHash","type":"bytes32"},{"name":"_signature","type":"bytes"}],"name":"isValidSignature","outputs":[{"name":"magicValue","type":"bytes4"}],"payable":False,"stateMutability":"view","type":"function"}]
```

Comment:
let's move this to the ABI folder 
https://github.com/gitcoinco/web/tree/master/app/grants/abi